static/src/scss/resources/_categories.scss

```diff
@@ -74,14 +75,31 @@
                 padding: 0.8rem 1.5rem;
             }
         }
+
+        .close-button-holder {
```

Comment:
let's add `cursor: pointer` to the actual clickable bit to give feedback to users that it does something